src/ncp/ncp_base.cpp

```diff
@@ -5167,6 +5169,7 @@ otError NcpBase::SetPropertyHandler_STREAM_NET_INSECURE(uint8_t aHeader, spinel_
 
     (void)aKey;
 
+	OT_UNUSED_VARIABLE(parsedLength);
```

Comment:
We have a `(void)parsedLength` along with `(void)metaPtr; (void)metaLen;`   few line above this. Do we still need this?